addons/mail/static/src/messaging.js

```diff
@@ -263,6 +275,15 @@ export class Messaging {
                         this.createMessage(body, message, thread);
                     }
                     break;
+                case "mail.record/insert":
```

Comment:
@seb-odoo @alexkuhn should we keep this notification name ? Doesn't make a lot of sens in the new code.

Reply:
There's no reason to remove, and actually there are very good design choices with a single notification format, so would keep it for the moment.